src/dbscan_spatial.cpp

```diff
@@ -308,7 +451,7 @@ class DbscanSpatial : public rclcpp::Node
   float minX = -80.0, minY = -25.0, minZ = -2.0;
   float maxX = +80.0, maxY = +25.0, maxZ = -0.15;
   double eps = 3.5;
-  bool verbose1 = false, verbose2 = false, pub_undecided = false;
+  bool verbose1 = false, verbose2 = true, pub_undecided = false; // to disable benchmarking, set verbose2 to false
```

Comment:
it is also possible to set in a launch file (without re-build)
https://github.com/jkk-research/lidar_cluster_ros2/blob/4bdd2d0e3c477929eec0752e987382fa4db9fce3/launch/dblane_spatial.launch.py#L22